### PR TITLE
Merge 4.10.0 into master

### PR DESCRIPTION
**Pull request by c-bordon**
related https://github.com/wazuh/wazuh-installation-assistant/issues/49

### CODE REVIEW

**Review by vcerenu — APPROVED** [review]

File: .github/actions/offline-installation/common.sh [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+
+}
+
+function download_resources() {
+
+    check_file "${ABSOLUTE_PATH}"/wazuh-install.sh
+    bash "${ABSOLUTE_PATH}"/wazuh-install.sh -dw "${sys_type}"
+    echo "INFO: Downloading the resources..."
+
+    curl -sO https://packages.wazuh.com/4.10/config.yml
```
Comment (teddytpc1) [inline .github/actions/offline-installation/common.sh:104]: Change the reference to `5.0`.

File: .github/actions/offline-installation/common.sh [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+    check_file "config.yml"
+
+    sed -i -e '0,/<indexer-node-ip>/ s/<indexer-node-ip>/127.0.0.1/' config.yml
+    sed -i -e '0,/<wazuh-manager-ip>/ s/<wazuh-manager-ip>/127.0.0.1/' config.yml
+    sed -i -e '0,/<dashboard-node-ip>/ s/<dashboard-node-ip>/127.0.0.1/' config.yml
+
+    curl -sO https://packages.wazuh.com/4.10/wazuh-certs-tool.sh
```
Comment (teddytpc1) [inline .github/actions/offline-installation/common.sh:111]: Change the reference to `5.0`.

**Review by teddytpc1 — CHANGES_REQUESTED** [review]

File: .github/actions/offline-installation/common.sh (hunk 1 above)
Comment (c-bordon) [reply .github/actions/offline-installation/common.sh]: Done 6aac622

File: .github/actions/offline-installation/common.sh (hunk 2 above)
Comment (c-bordon) [reply .github/actions/offline-installation/common.sh]: Done 6aac622

**Review by teddytpc1 — APPROVED** [review]

**Pull request merged**